docs/src/pages/[platform]/theming/theme-provider/index.page.mdx

```diff
@@ -1,6 +1,8 @@
 ---
 title: ThemeProvider
-description: The ThemeProvider allows you to apply a Theme to your application.
+description: The ThemeProvider allows you to apply a Theme to your application. ThemeProvider used to be called AmplifyProvider. The ThemeProvider
```

Comment:
Should this be in the metaDescription instead of the description?

Reply:
I wonder if metaDescription would be crawled since it's not visible to the users 🤔

Reply:
Just made it shorter

Reply:
Used an Alert instead to match the rest of our site.